main.js

```diff
@@ -195,6 +196,21 @@ client.on('ready', async () => {
   console.log(chalk.greenBright('Bot online and Ready!'));
 });
 
+// Anti-swear system
+client.on("messageCreate", async message => {
+  if (message.channel.type === "DM") return // Do not send messages in DMs
+  if (message.deleted) return // If the message was deleted, then return
+  if (!message.member.permissions.has(Permissions.FLAGS.BAN_MEMBERS)) { // Should be made configurable soon
+    if (filter.check(message.content)) {
+      message.delete()
+      const warning = await message.channel.send(
+        `${message.author}, do not use bad words in chat!`
+      )
+      setTimeout(warning.delete, 4500)
+    }
+  }
+})
+
```

Comment:
Hey! This should live in a file inside of the features directory.

Reply:
> Hey! This should live in a file inside of the features directory.

Alright! I'll work on that, thanks. 🙂